src/MakaMek.MapEditor.Android/README.md

```
@@ -0,0 +1,33 @@
+ # MakaMek.MapEditor.Android
```

markdownlint-cli2 (0.23.2)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)